### PR TITLE
Fix minor issues in `iCubGazeboV3_visuomanip`

### LINKED ISSUE #169: Create iCubGazeboV3_visuomanip
For simulations related to teleoperation, it would be useful to have an iCubGazeboV3 model with moving eyes, cameras and moving hands. However, in the short term it would be difficult to obtain this information directly from the CAD, so probably we need to create this model by taking the hands and he head from the `[iCubGazeboV2_5_visuomanip](https://github.com/robotology/icub-models/tree/master/iCub_manual/conf_manual/iCubGazeboV2_5_visuomanip)`, and adding them to the existing `iCubGazeboV3` model. 

Related issues:
* https://github.com/robotology/icub-models/pull/42
* https://github.com/robotology/icub-models-generator/issues/37
* https://github.com/robotology/icub-models-generator/issues/29

cc @S-Dafarra

### PR DESCRIPTION
**Pull request by mebbaid**
This is a PR that fixes
- `r_hand` collision parameters. reference issue #169 
- eyes pan joints poses. reference issue #https://github.com/ami-iit/component_ANA-Avatar-XPRIZE/issues/726
- eyes camera sensors and corresponding configurations. reference issue #https://github.com/ami-iit/component_ANA-Avatar-XPRIZE/issues/726

When finialized, this PR should close #169

### CODE REVIEW

**Comment by mebbaid** [comment]
I will squash, and cancel irrelevant commits and select only merged commits.

**Comment by xEnVrE** [comment]
Hi @mebbaid,

if I a not wrong this PR is also changing the files associated to the `iCubGazeboV2_5_visuomanip` model. Is this a mistake or is it intended? I am referring to [these](https://github.com/robotology/icub-models/pull/176/files#diff-c7019e6f666f091be38c940cc5c01e193b6384e896fa2396f691b1eb4d30a4c9) changes.

Thank you

**Comment by mebbaid** [comment]
> Hi @mebbaid,
> 
> if I am not wrong this PR is also changing the files associated to the `iCubGazeboV2_5_visuomanip` model. Is this a mistake or is it intended? I am referring to [these](https://github.com/robotology/icub-models/pull/176/files#diff-c7019e6f666f091be38c940cc5c01e193b6384e896fa2396f691b1eb4d30a4c9) changes.
> 
> Thank you

Thanks for the catch @xEnVrE , indeed these changes are probably a copy mistake. Will discard them.

**Comment by traversaro** [comment]
@mebbaid can you rebase on top of master? It seems that there is the commit https://github.com/robotology/icub-models/pull/176/commits/f547a99c7709cd8eba0f6240b72de0c1363ce2a8 in this release that actually already belongs to master.

**Comment by mebbaid** [comment]
> Hi @mebbaid,
> 
> if I am not wrong this PR is also changing the files associated to the `iCubGazeboV2_5_visuomanip` model. Is this a mistake or is it intended? I am referring to [these](https://github.com/robotology/icub-models/pull/176/files#diff-c7019e6f666f091be38c940cc5c01e193b6384e896fa2396f691b1eb4d30a4c9) changes.
> 
> Thank you

Hi @xEnVrE , I was indeed confused why the file changed since the output of `git log --follow iCub_manual/robots/iCubGazeboV2_5_visuomanip/model.urdf` gives me the following 


```
(base) mebbaid@IITICUBLAP213:~/robot-code/mebbaid_icub-models/icub-models$ git log --follow iCub_manual/robots/iCubGazeboV2_5_visuomanip/model.urdf
commit 0a00225459d35eb45e8be676388ee906c4dc2cb1
Author: Nicola Piga <nicolapiga@gmail.com>
Date:   Mon Mar 14 11:28:51 2022 +0100

    Fix wrong collision of 'l_hand_little_1' in iCub_manual/iCubGazeboV2_5

commit 6f1d59b59aaa024dc9117c6523e22c8baaf05f26
Author: Nicola Piga <nicolapiga@gmail.com>
Date:   Fri Mar 11 17:43:17 2022 +0100

    Fix material colors of iCub_manual/iCubGazeboV2_5_visuomanip as per https://github.com/robotology/icub-models-generator/pull/217

commit ac14772fb48d8aa887bc3b2397491e5762749745
Author: Ugo Pattacini <ugo.pattacini@iit.it>
Date:   Sun Jul 26 16:28:40 2020 +0200

    address #44

commit e0b88c461e7996003bdfe340cef9ea741f3ecf6c
Author: Nicola Piga <nicolapiga@gmail.com>
Date:   Thu Jul 23 17:11:30 2020 +0200

    Add model iCub_manual/iCubGazeboV2_5_visuomanip
```

with no local changes. Hopefully things should be ok now.

File: iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_rgb_camera.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[include "gazebo_icub_robotname.ini"]
+
+device grabberDual
```
Comment (Nicogene) [inline iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_rgb_camera.ini:3]: Not related to this PR specifically, but I would avoid using devices that are deprecated, could you please use `frameGrabber_nws_yarp` instead?
Thanks!

**Review by Nicogene — CHANGES_REQUESTED** [review]
My comments are related to the network wrappers used, in general we should port all the models to the new one for supporting future versions of YARP

cc @traversaro

File: iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_rgb_camera.ini (hunk 1 above)
Comment (mebbaid) [reply iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_rgb_camera.ini]: Thanks for the heads up about the cameras deprecated devices. Done in https://github.com/mebbaid/icub-models/commit/91a5d1c4a362faafeb5498f4effb400d6f3b30d3 and tested.

File: iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_right_depth_camera.ini [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[include "gazebo_icub_robotname.ini"]
+
+device RGBDSensorWrapper
```
[the remark this answers is not in the document]
Comment (mebbaid) [reply iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_right_depth_camera.ini]: Thanks for the heads up about the cameras deprecated devices. Done in https://github.com/mebbaid/icub-models/commit/91a5d1c4a362faafeb5498f4effb400d6f3b30d3 and tested.

**Review by Nicogene — APPROVED** [review]

**Review by traversaro — APPROVED** [review]

**Pull request merged**